### DISCUSSION TITLE
same data in diff collection has diff results

### DISCUSSION THREAD

**Discussion by bingbb47** [milvus-io/milvus#19669] [Q&A and General discussion]
when I search on the milvus2.1.2，strange results appear, the distance and rank not match
![image](https://user-images.githubusercontent.com/111261344/194790541-f33ef4cf-d59c-4fe2-a538-21c0a3d5d85d.png)
and i create a new collection and insert the same vector, do search again,  the result is rationable
![image](https://user-images.githubusercontent.com/111261344/194790715-60bcdc76-b8ef-42ca-bad0-639eb5fb6976.png)
I can't trace what happend on that collection, so do you have any idea about it?

**Comment by bingbb47** [comment]
the metric is IP

**Comment by yhmo** [comment]
Do you have a script to reproduce this?

**Reply by bingbb47** [reply]
No, i have no idea to reproduce the collection now. Can you guess the reasons for this phenomenon? We can see the rank of ids is same but distance are weird.(and are the results only sorted by distance? The results are not strictly descending in the left rank)
![image](https://user-images.githubusercontent.com/111261344/195238025-e4149369-8090-43b6-b815-403aa23d144a.png)
(sry,wrong pictures were uploaded before)

the differents of original collection and backup collection are:
1. the origin is Strong ;the backup is Bounded
2. the origin have created FLAT index by JAVA sdk; the backup not
3. the origin may occurred deletion ,because the number of query by "id > 0" less than num_entities
